support/ext/event

```diff
@@ -6,6 +6,10 @@ php_version=${1:=7.4}
 
 event_version="3.0.5"
```

Comment:
Can  you move this version on `conf/buildpack.conf` too and source the file ?

Reply:
All versions in `support/ext/*` are in the script file. These extensions are different than the extensions in `support/ext-internal`. Internal extensions are more linked to PHP and their versions are in `conf/buildpack.conf` because we also need these version numbers in `lib/composer`.

Reply:
Oh okay, thanks for the clarification 😃